editor/src/components/inspector/controls/color-picker.tsx

```diff
@@ -322,16 +338,21 @@ export class ColorPickerInner extends React.Component<
   }
 
   submitNewColor = (chromaColor: Chroma.Color, transient: boolean) => {
-    const newValue = toPassedInColorType(
-      chromaColor,
-      this.props.value,
-      this.state.normalisedHuePosition,
-    )
-    if (transient) {
-      this.props.onTransientSubmitValue(newValue)
-    } else {
-      this.props.onSubmitValue(newValue)
+    if (this.updateTimeoutId.current != null) {
```

Comment:
this will only update the color on the canvas once you let go of dragging, right?

Reply:
Nope, if you stop dragging for a moment it'll fire the update as well.

Reply:
I updated my comment, I think we should use onTransientSubmitValue here